R/LSBuffonsneedlemanipulation.R

```diff
@@ -95,8 +95,16 @@ LSBuffonsneedlemanipulation   <- function(jaspResults, dataset, options, state =
     propDistPlot$plotObject <- jaspGraphs::themeJasp(propDistPlot0)
     
     if (options[["legendPropDistPlot"]]){
-      propDistPlot$plotObject <-  propDistPlot$plotObject + 
-        ggplot2::theme(legend.position = c(.17, .9))
+      if (xValue[which.max(propPost)] < 0.5){
+        propDistPlot$plotObject <-  propDistPlot$plotObject + 
+          ggplot2::theme(legend.position = c(.75, 1))
+      }
+      
+      if (xValue[which.max(propPost)] >= 0.5){
+        propDistPlot$plotObject <-  propDistPlot$plotObject + 
+          ggplot2::theme(legend.position = c(.25, 1))
+      }
```

Comment:
```suggestion
      propDistPlot$plotObject <- propDistPlot$plotObject +
        ggplot2::theme(legend.position="right")
```

Move the legend entirely out of the plot area, which makes your life easier, and simplifies the code a lot.

Do this for every plot with a legend please.